setup.py

```diff
@@ -45,7 +45,7 @@
     packages=find_namespace_packages(include=["pytket.*"]),
     include_package_data=True,
     install_requires=[
-        "pytket >= 2.2.0",
+        "pytket >= 2.4.0rc0",
```

Comment:
I'm not sure we should merge this until we have a proper release we can depend on.

Reply:
I would suggest we merge this now, but wait with a release?